src/runtime_src/core/pcie/tools/xbmgmt/cmd_config.cpp

```diff
@@ -28,7 +28,7 @@
 const char *subCmdConfigDesc = "Parse or update daemon/device configuration";
 const char *subCmdConfigUsage =
     "--daemon --host ip-or-hostname-for-peer\n"
-    "--device [--card bdf] [--security level] [--runtime_clk_scale en(dis)able]\n"
+    "--device [--card bdf] [--security level] [--runtime_clk_scale en(dis)able] [--csThresholdPowerOverride val]\n"
```

Comment:
please add new options in similar way as existing ones. you have --runtime_clk_scale already, so you probably don't want to use "cs" and should use "_", instead of upper case letters.